### ISSUE TITLE
don't use color scheme font?

### ISSUE THREAD

**Issue by boneskull** [rainglow/jetbrains#11]
Would you be interested in removing the font from the color scheme?  I am using my own font, and I have to deselect "Use color scheme font instead of the default" for each Rainglow scheme.

**Comment by bozhkos** [comment]
I asked about that before, and ended up exporting favorite themes, with few changed options into separate themes #10

**Comment by voydz** [comment]
I second that, 

I think that **color* schemes should not interfere with your font settings.

**Comment by daylerees** [comment]
I'll take a look guys. Knowing it could be overriden I thought a default would be nice but I'm happy to remove it if people feel that's not the case!

**Comment by perlun** [comment]
I agree with previous sentiments in this thread; it breaks the [principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) and the default doesn't even look very good on my screen. :wink: Of course that's a matter of personal preference, but I would also prefer it if the color schemes would _only_ touch the colors, not the fonts.

**Comment by Lazzu** [comment]
I agree with this. The current CLion just tells the user that the color theme is overriding the font settings, and I have to change the theme settings to not override it.
